basic-auth/README.md

```diff
@@ -0,0 +1,67 @@
+# Basic Authentication
+
+
+This sample allows you to authenticate an incoming request with a Basic Authentication header using a `USER_ID` and a `USER_PASSWORD` as the encoded credential pair. 
+
+## Prerequisites
+1. [Provision Apigee X](https://cloud.google.com/apigee/docs/api-platform/get-started/provisioning-intro)
+2. Configure [external access](https://cloud.google.com/apigee/docs/api-platform/get-started/configure-routing#external-access) for API traffic to your Apigee X instance
+3. Access to deploy proxies, create products, apps and developers in Apigee
+4. Make sure the following tools are available in your terminal's $PATH (Cloud Shell has these preconfigured)
+   * [gcloud SDK](https://cloud.google.com/sdk/docs/install)
+   * unzip
+   * curl
+   * jq
+   * npm
+
+# (QuickStart) Setup using CloudShell
+
+Use the following GCP CloudShell tutorial, and follow the instructions.
+
+[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.png)](https://ssh.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/apigee-samples&cloudshell_git_branch=main&cloudshell_workspace=.&cloudshell_tutorial=basic-auth/docs/cloudshell-tutorial.md)
+
+## Setup instructions
+
+1. Clone the apigee-samples repo, and switch the basic-auth directory
+
+```bash
+git clone https://github.com/GoogleCloudPlatform/apigee-samples.git
+cd apigee-samples/basic-auth
+```
+
+2. Edit the `env.sh` and configure the ENV vars. If you don't set variables that are marked as `Optional`, then the deploy script will also provision a mock OIDC authorization server that will allow you to issue JWT access tokens to test this sample
```

Comment:
need to rewrite this (looks like a paste from the other sample :) )

Reply:
fixed.